### ISSUE TITLE
Is our app aimed to teenagers in general or teenagers with low self-esteem specifically?

### ISSUE THREAD

**Issue by marisid** [CYPIAPT-LNDSE/social-up#19]
We were wondering if the app and its content should more widely open to turning negative feelings to positive emotions following the CBT technique? 

Or is it rather aimed only for young people with low self-esteem, in which case the users should be more carefully selected for the user testing period?

We need to understand our typical user profile better in order to create a product of value. We were thinking more of a general CBT app (e.g. teenagers may feel angry because they read a post about the shutdown of pub/club they really liked and they don't necessarily feel self-conscious). 

@MatteoTurco could you provide feedback on this please? That would also determine the content (images/text/CBT exercises/ fact sheet etc) and wording we choose for the app, even in the design phase we are at the moment.

**Comment by MatteoTurco** [comment]
Focusing on the target audience is an excellent point. I like the idea of a software that is more widely applicable, as suggested. And I agree that it should appeal to the widest pool of youngster possible.
Having said that, I wonder about what is feasible. My understanding of CBT techniques (of which there's a wide variety, with multiple applications) is that they are rather focused, in the sense that they are based on theories of psychological function/dysfunction and target specific aspects of cognition. For example, a CBT technique designed for general anxiety aims at breaking a specific cognitive cycle of avoidance; or as in our original thinking ground, it may aim at morphing negative cyclical thinking patterns that relate to the self into more positive ones - which in turn raises self-esteem (these are of course reductionistic views, just to outline the idea). A person with no issues of self-esteem may very well find a picture of a friend on a nice holiday jealousy-provoking, but that doesn't mean it will lower his or her self-esteem. Self-esteem relies on perceptions of the self that are, again, simplistically speaking, moulded by how we speak to ourselves. It is not merely influenced by external stimuli. 
I think it's important to be mindful that CBT techniques do not simply change a negative thought into a positive one. As nice as that may sound, unfortunately it doesn't works that way. It would be tricky to try and target everything under the sun with what are rather focused techniques. I acknoweldge that this is a personal opinion, so I will think it over with someone more knowledgeable than myself - I will speak to my colleagues and bosses tomorrow.
If we are interested in a wider application, we could go down the route of also employing meditative/mindful techniques adapted for psychological needs, as these can theoretically be used by a wider pool of people. 
In short I would say that our typical user is someone susceptible to low self-esteem, and we should target that susceptibility, rather than targeting the virtually infinite array of negative emotions that can stem from it.
Please let me know what you think.
